packages/vuepress-theme-odyssey/components/DocsCard.vue

```diff
@@ -0,0 +1,42 @@
+<template>
+  <div
```

Comment:
`article` maybe? Or `li` when used as a list item

Reply:
Added a `tag` prop to `DocsCard` and `DocsCardGroup` which allows you to change the tag. This is now a `ul`/`li` as it is a list.